tools/lib/provision.py

```diff
@@ -317,7 +317,12 @@ def main(options):
         print("No need to run `tools/setup/build_pygments_data`.")
 
     run(["scripts/setup/generate_secrets.py", "--development"])
-    run(["tools/update-authors-json", "--use-fixture"])
+
+    update_authors_json_paths = ["zerver/tests/fixtures/authors.json"]
```

Comment:
`tools/update-authors-json` itself should be in the list; otherwise we won't re-provision when that file's code changes.